2-mandatory/2-currency-code-lookup.js

```diff
@@ -18,8 +18,13 @@ const COUNTRY_CURRENCY_CODES = [
 ];
 
 function createLookup(countryCurrencyCodes) {
-  // write code here
+  const lookup = {};
+  for(const [countryCode, currencyCode] of countryCurrencyCodes){
+    lookup[countryCode] = currencyCode;
+  }
+  return lookup
 }
+console.log(createLookup(COUNTRY_CURRENCY_CODES));
 
```

Comment:
Or we can write the code as following:
function createLookup(countryCurrencyCodes) {
  let currencyCodeLookup = {};

  for (let countryCurrencyCode of countryCurrencyCodes) {
    let countryCode = countryCurrencyCode[0];
    let currencyCode = countryCurrencyCode[1];
    currencyCodeLookup[countryCode] = currencyCode;
  };

  return currencyCodeLookup;
}

Reply:
I think your original solution is better, but this other solution is also nice and clear.